src/igvfd/searches/configs/analysis_set.py

```diff
@@ -44,13 +44,13 @@ def analysis_set():
                 'title': 'Treatment'
             },
             'construct_library_sets.file_set_type': {
-                'title': 'Construct Library Data'
+                'title': 'Construct Library'
             },
             'construct_library_sets.selection_criteria': {
                 'title': 'Construct Library Selection Criteria'
             },
-            'construct_library_sets.integrated_content_files.content_type': {
-                'title': 'Construct Library Design'
+            'construct_library_sets.associated_phenotypes.term_name': {
```

Comment:
did you intend to remove `construct_library_sets.integrated_content_files.content_type`? Asking because our last wrangler meeting, you argued to keep it. The reasoning was that there is interest in know what's in the inserts/ what's in the cloned in vectors.